R/process_observers.R

```diff
@@ -505,18 +527,23 @@ get_observer_quality <- function(mbbs_survey_events) {
         which.max(c(.data$obs1_quality, .data$obs2_quality, .data$obs3_quality)),
       observer_quality = case_when(
         # if there's only one observer
-        (sum(is.na(c(.data$obs1, .data$obs2, .data$obs3)) == FALSE) == 1) ~ .data$observer_quality, # don't change obs_quality
+        # don't change obs_quality
+        (sum(is.na(c(.data$obs1, .data$obs2, .data$obs3)) == FALSE) == 1) ~ .data$observer_quality,
         # obs 1 is the best observer but only has one survey across all routes
-        (max_qual_observer == 1 & obs1_nsurveys == 1) ~ suppressWarnings(max(obs2_quality, obs3_quality, na.rm = TRUE)), # take max of obs2 and obs3
+        # take max of obs2 and obs3
+        (max_qual_observer == 1 & obs1_nsurveys == 1) ~ suppressWarnings(max(obs2_quality, obs3_quality, na.rm = TRUE)),
         # obs 2 is the best observer but only has one survey across all routes
-        (max_qual_observer == 2 & obs2_nsurveys == 1) ~ suppressWarnings(max(obs1_quality, obs3_quality, na.rm = TRUE)), # take max of obs1 and obs3
+        # take max of obs1 and obs3
+        (max_qual_observer == 2 & obs2_nsurveys == 1) ~ suppressWarnings(max(obs1_quality, obs3_quality, na.rm = TRUE)),
         # obs 3 is the best observer but only has one survey across all routes
-        (max_qual_observer == 3 & obs3_nsurveys == 1) ~ suppressWarnings(max(obs1_quality, obs2_quality, na.rm = TRUE)), # take max of obs1 and obs2
+        # take max of obs1 and obs2
+        (max_qual_observer == 3 & obs3_nsurveys == 1) ~ suppressWarnings(max(obs1_quality, obs2_quality, na.rm = TRUE)),
         # if none of the other statements are true, leave obs_quality the same
         TRUE ~ .data$observer_quality
       )
     ) %>%
     ungroup()
 
-  return(mbbs_survey_events)
+  # return
+  mbbs_survey_events
```

Comment:
Yeah like this

Reply:
got it! I removed many of them because the last thing that had anything assigned to it was the same thing I intended to return. I'll add it back in